daemon/openrazer_daemon/hardware/accessory.py

```diff
@@ -68,3 +68,43 @@ class RazerBaseStationChroma(_RazerDeviceBrightnessSuspend):
     RAZER_URLS = {
         "top_img": DEVICE_IMAGE
     }
+
+
+class RazerNommoChroma(_RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Nommo Chroma (Speakers)
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x0517
+    HAS_MATRIX = True
+    MATRIX_DIMS = [2, 24]
+    METHODS = ['get_device_type_accessory', 'set_static_effect', 'set_wave_effect', 'set_spectrum_effect',
+               'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row']
+
+    DEVICE_IMAGE = "https://images-na.ssl-images-amazon.com/images/I/51wEJVJjuLL._AC_SL1000_.jpg"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": DEVICE_IMAGE
+    }
+
+
+class RazerNommoPro(_RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Nommo Pro (Speakers)
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x0518
+    HAS_MATRIX = True
+    MATRIX_DIMS = [2, 8]
+    METHODS = ['get_device_type_accessory', 'set_static_effect', 'set_wave_effect', 'set_spectrum_effect',
+               'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row']
+
+    DEVICE_IMAGE = "https://images-na.ssl-images-amazon.com/images/I/51GXgPtBB0L._AC_SL1000_.jpg"
```

Comment:
And https://d4kkpd69xt9l7.cloudfront.net/sys-master/root/he6/h1a/8939927175198/nommo-gallery-1500x1000-11.jpg here

Reply:
Those are the same picture.  Did you paste the wrong image for the Nommo Pro?

Reply:
How about this picture for Nommo Pro:

https://d4kkpd69xt9l7.cloudfront.net/sys-master/root/h89/hc4/9003754717214/razer-nommo-pro-audio-04.jpg